integration_tests/tests/temporary-accommodation/manage/lostBed.cy.ts

```diff
@@ -0,0 +1,253 @@
+import BedspaceShowPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/bedspaceShow'
+import Page from '../../../../cypress_shared/pages/page'
+import LostBedNewPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/lostBedNew'
+import bedFactory from '../../../../server/testutils/factories/bed'
+import LostBedShowPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/lostBedShow'
+
+context('Lost bed', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    setupTestUser()
+  })
+
+  it('navigates to the create void bedspace page', () => {
+    // Given I am signed in
+    cy.signIn()
+
+    // And there is a premises and a room the database
+    const premises = premisesFactory.build()
+    const room = roomFactory.build()
+
+    cy.task('stubSinglePremises', premises)
+    cy.task('stubSingleRoom', { premisesId: premises.id, room })
+
+    // When I visit the show bedspace page
+    const bedspaceShow = BedspaceShowPage.visit(premises, room)
+
+    // Add I click the void bedspace link
+    cy.task('stubLostBedReferenceData')
+    bedspaceShow.clickVoidBedspaceLink()
+
+    // Then I navigate to the new lost bed page
+    Page.verifyOnPage(LostBedNewPage)
+  })
+
+  it('navigates to the show void bedspace page', () => {
+    // Given I am signed in
+    cy.signIn()
+
+    // And there is a premises, a room, and lost beds the database
+    const premises = premisesFactory.build()
+    const room = roomFactory.build()
+    const lostBeds = lostBedFactory
+      .active()
+      .params({
+        bed: bedFactory.build({
+          id: room.beds[0].id,
+        }),
+      })
+      .buildList(5)
+
+    cy.task('stubSinglePremises', premises)
+    cy.task('stubSingleRoom', { premisesId: premises.id, room })
+    cy.task('stubLostBedsForPremisesId', { premisesId: premises.id, lostBeds })
+    cy.task('stubSingleLostBed', { premisesId: premises.id, lostBed: lostBeds[0] })
+
+    // When I visit the show bedspace page
+    const bedspaceShowPage = BedspaceShowPage.visit(premises, room)
+
+    // Add I click the lost bed link
+    bedspaceShowPage.clickLostBedLink(premises.id, room.id, lostBeds[0].id)
+
+    // Then I navigate to the show lost bed page
+    Page.verifyOnPage(LostBedShowPage, premises, room, lostBeds[0])
+  })
+
+  it('allows me to create a void booking', () => {
+    // Given I am signed in
+    cy.signIn()
+
+    // And there is a premises and a room the database
+    const premises = premisesFactory.build()
+    const room = roomFactory.build()
+
+    cy.task('stubSinglePremises', premises)
+    cy.task('stubSingleRoom', { premisesId: premises.id, room })
+
+    // When I visit the void a bedspace page
+    cy.task('stubLostBedReferenceData')
+    const page = LostBedNewPage.visit(premises.id, room.id)
+
+    // And I fill out the form
+    const lostBed = lostBedFactory.build({
+      bed: bedFactory.build({
+        id: room.id,
```

Comment:
My instinct is something's gone amiss if we're assigning the room ID to the bed ID. Is this fixed by the change I suggested to `new.njk`?

Reply:
yes, good spot